lib/ex_aws/s3.ex

```diff
@@ -684,24 +684,38 @@ defmodule ExAws.S3 do
         # Uncomment if you need to gunzip (and add dependency :stream_gzip)
         # |> StreamGzip.gunzip()
         |> Stream.chunk_while("", &chunk_fun/2, &after_fun/1)
```

Comment:
It looks like the `after_fun` above be cahnged to `to_line_stream_after_fun`.

Reply:
Good catch, I have corrected the function name here.